src/notebooks/controllers/installPythonControllerCommands.ts

```diff
@@ -179,9 +183,10 @@ export class InstallPythonControllerCommands implements IExtensionSingleActivati
                 // Don't move forward until we have hooked the API
                 // Note extensions.installExtension seems to return "mostly" after the install is done, but at that
                 // point we don't see it installed via the checker and don't have the API so wait for it here
-                await this.pythonApi.pythonExtensionHooked;
+                const hookResult = await Promise.race([sleep(60_000), this.pythonApi.pythonExtensionHooked]);
```

Comment:
The timeout is a bit long here, but that was intentional. After VSCode returns from the .installExtension command it doesn't actually show up as installed yet. We need to wait a bit for it to actually get registered and activated. Since python takes a while to activate made a longer wait here. On the plus, it's just activation and not all interpreter discovery.

Reply:
Is there a way to tell if something is disabled instead of not installed?

Reply:
Ideally we'd skip installing and instead we'd ask to enable.